sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/Subscriber.groovy

```diff
@@ -0,0 +1,30 @@
+package life.qbic.business.project.subscribe
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
+class Subscriber {
```

Comment:
this information could be taken from liferay as well, but I don't have strong feelings about using a class